cassdegrees/static/js/staff/globalrequirements.js

```diff
@@ -83,22 +83,23 @@ Vue.component('global_requirement_general', {
         const request = new XMLHttpRequest();
 
         request.addEventListener("load", () => {
-            rule.check_options();
+            rule.check_options(false);
         });
         request.open("GET", "/api/search/?select=code&from=course");
         request.send();
     },
     methods: {
-        check_options() {
+        check_options(is_submission) {
             this.invalid_units = this.details.unit_count <= 0;
             this.invalid_units_step = this.details.unit_count % 6 !== 0;
             this.units_is_blank = this.details.unit_count === "";
```

Comment:
These are set whether or not `is_submission` is true, meaning errors can be introduced at any time. This goes against what the PR is saying it accomplishes

Reply:
forgot to say in the PR, this is intentional. the reason for this is we dont really want several validation messages popping up at once. The ones that will only pop up when they try to submit are the ones that were instantly popping up.